File: src/Classes/View/NavItems/HamburgerItem.php
<?php
namespace MDJ\View\NavItems;

use MDJ\Interfaces\iView;
// use MDJ\src\Config;

class HamburgerItem extends NavItem 
{
    public function show() 
	{
        echo '<li class="'.$this->li_class.'"> 
                    <a class="px-3 '.$this->a_class.'" href="'.\Config::BASEURL.'?page='.$this->page.'">'.$this->name.'</a>
              </li>';
    }
}

File: ArticleReviewApp/ArticleReviewApp/Models/HamburgerItem.cs
using System;
using System.Collections.Generic;

namespace ArticleReviewApp.Models;

public partial class HamburgerItem
{
    public string Page { get; set; } = null!;

    public string Name { get; set; } = null!;

    public string LiClass { get; set; } = null!;

    public string AClass { get; set; } = null!;
}
